### PR TITLE
Update vagrant setup

### PR DESCRIPTION
**Pull request by xtreme-vikram-yadav**
Provide user with vagrant file and bootstrap script for setup to save time when running tern.

Signed-off-by: Yoon Jean Kim <ykim@pivotal.io>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,5 @@ report.yml
 report.json
 tern.log
 cache.yml
+vagrant/.*
+.idea
```
Comment (nishakm) [inline .gitignore]: I am not familiar with the .idea file/folder. Is it required for vagrant to work?

Comment (xtreme-vikram-yadav) [reply .gitignore]: Hey @nishakm , .idea folder is IDE specific folder created by intellij IDEs to store project related stuff.

Comment (nishakm) [reply .gitignore]: @xtreme-vikram-yadav since this is IDE specific, could you leave it out of the .gitignore?

Comment (xtreme-vikram-yadav) [reply .gitignore]: @nishakm it's a minor thing but this normally shouldn't be committed to this project and will make it convenient for contributors to look at tracked files

Comment (xtreme-vikram-yadav) [reply .gitignore]: @nishakm Update should be in

Comment (nishakm) [reply .gitignore]: @xtreme-vikram-yadav Thank you! This looks good.

**Pull request merged**